lib/binding_web/src/marshal.ts

```diff
@@ -168,10 +168,13 @@ export function marshalEdit(edit: Edit, address = TRANSFER_BUFFER) {
  *
  * Unmarshals a {@link LanguageMetadata} from the transfer buffer.
  */
-export function unmarshalLanguageMetadata(address: number): LanguageMetadata {
-  const result = {} as LanguageMetadata;
-  result.major_version = C.getValue(address, 'i32'); address += SIZE_OF_INT;
-  result.minor_version = C.getValue(address, 'i32'); address += SIZE_OF_INT;
-  result.field_count = C.getValue(address, 'i32');
```

Comment:
I think this was an error because `field_count` doesn't exist in `LanguageMetadata`, I have changed it to `patch_version`.

Reply:
:facepalm: thank you